summary-of-summaries/src/main.ts

```diff
@@ -67,23 +58,16 @@ button.addEventListener('click', async () => {
 });
 
 const checkSummarizerSupport = async (): Promise<boolean> => {
-    // Checks availability against the new API shape.
-    if (self.Summarizer !== undefined) {
-      let availability = await window.Summarizer.availability();
-      return availability === 'available' || availability === 'downloadable';
-    }
-  
-    // Checks availability agains the old API shape.
-    let capabilities = await window.ai.summarizer.capabilities();
-    return capabilities.available === 'readily' || capabilities.available === 'after-download';  
-  }
+    // Checks model availability.
+    let availability = await window.Summarizer.availability();
+    return availability === 'available' || availability === 'downloadable';
+ }
 
-  if (self.Summarizer || (window.ai && window.ai.summarizer)) {
+  if (self.Summarizer) {
```

Comment:
```suggestion
  if (window.Summarizer) {
```

Mostly for consistency.